fontir/src/ir.rs

```diff
@@ -4,8 +4,8 @@ use crate::{
     coords::{CoordConverter, NormalizedCoord, NormalizedLocation, UserCoord},
     error::{PathConversionError, VariationModelError, WorkError},
     serde::{
-        deserialize_name_id, deserialize_tag, serialize_name_id, serialize_tag,
-        GlobalMetricsSerdeRepr, GlyphSerdeRepr, StaticMetadataSerdeRepr,
+        deserialize_name_id, serialize_name_id, GlobalMetricsSerdeRepr, GlyphSerdeRepr,
```

Comment:
Can we drop the extra serialization support for name id as well?

Reply:
I believe we can!